Description: Upstream changes introduced in version 1:1.2.3.4.dfsg-3stormos0
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 zlib (1:1.2.3.4.dfsg-3stormos0) unstable; urgency=low
 .
   * Package for StormOS.
 .
 The person named in the Author field signed this changelog entry.
Author: Andrew Stormont (astormont) <andyjstormont@googlemail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- zlib-1.2.3.4.dfsg.orig/zutil.h
+++ zlib-1.2.3.4.dfsg/zutil.h
@@ -37,6 +37,10 @@
 #   include <errno.h>
 #endif
 
+#ifdef __sun
+#include <sys/types.h> 
+#endif
+
 #ifndef local
 #  define local static
 #endif
